core/src/test/scala/unit/kafka/server/ShareGroupHeartbeatRequestTest.scala

```diff
@@ -592,6 +592,81 @@ class ShareGroupHeartbeatRequestTest(cluster: ClusterInstance) {
     }
   }
 
+  @ClusterTest(
+    types = Array(Type.KRAFT),
+    serverProperties = Array(
+      new ClusterConfigProperty(key = "group.share.enable", value = "true"),
```

Comment:
When you merge the latest changes, you'll find that `group.share.enable` is not required.